### PR TITLE
Fix misleading credential of MongoDB

### PR DESCRIPTION
**Pull request by ebyhr**
Related to https://stackoverflow.com/questions/59360176/how-to-connect-presto-to-mongodb-container/

### CODE REVIEW

File: presto-mongodb/src/main/java/io/prestosql/plugin/mongodb/MongoClientConfig.java [hunk 1]
```diff
@@ -129,16 +129,16 @@ public MongoClientConfig setCredentials(String... credentials)
         ImmutableList.Builder<MongoCredential> builder = ImmutableList.builder();
         for (String userPassCollection : userPasses) {
             int lastIndex = userPassCollection.lastIndexOf('@');
-            checkArgument(lastIndex > 0, "Invalid Credential format. Requires user:password@collection");
+            checkArgument(lastIndex > 0, "Invalid Credential format. Requires user:password@authentication-database");
             String userPass = userPassCollection.substring(0, lastIndex);
-            String collection = userPassCollection.substring(lastIndex + 1);
+            String database = userPassCollection.substring(lastIndex + 1);
```
Comment (findepi) [inline presto-mongodb/src/main/java/io/prestosql/plugin/mongodb/MongoClientConfig.java]: Also rename `userPassCollection` ?

File: presto-docs/src/main/sphinx/connector/mongodb.rst [hunk 2]
```diff
@@ -72,7 +72,7 @@ This property is optional; the default is ``_schema``.
 ``mongodb.credentials``
 ^^^^^^^^^^^^^^^^^^^^^^^
 
-A comma separated list of ``username:password@collection`` credentials.
+A comma separated list of ``username:password@authentication-database`` credentials.
```
Comment (findepi) [inline presto-docs/src/main/sphinx/connector/mongodb.rst]: `authentication-database` or just `database`?
(we provide this value to `com.mongodb.MongoCredential#createCredential` which calls this just "database", but we're free to use a different wording in docs, if necessary)

is it obvious what "authentication-database" stands for, or does it require additional explanation?

Comment (ebyhr) [reply presto-docs/src/main/sphinx/connector/mongodb.rst]: While I feel authentication-database is clear to most people, wrote additional explanation for quickstart.

File: presto-mongodb/src/main/java/io/prestosql/plugin/mongodb/MongoClientConfig.java [hunk 3]
```diff
@@ -127,18 +127,18 @@ public MongoClientConfig setCredentials(String... credentials)
...
     {
         ImmutableList.Builder<MongoCredential> builder = ImmutableList.builder();
-        for (String userPassCollection : userPasses) {
-            int lastIndex = userPassCollection.lastIndexOf('@');
-            checkArgument(lastIndex > 0, "Invalid Credential format. Requires user:password@collection");
-            String userPass = userPassCollection.substring(0, lastIndex);
-            String collection = userPassCollection.substring(lastIndex + 1);
+        for (String userPassDatabase : userPasses) {
+            int lastIndex = userPassDatabase.lastIndexOf('@');
+            checkArgument(lastIndex > 0, "Invalid Credential format. Requires user:password@authentication-database");
```
Comment (findepi) [inline presto-mongodb/src/main/java/io/prestosql/plugin/mongodb/MongoClientConfig.java]: `authentication-database` -> `database` ?

(same below)

**Pull request merged**